app/src/main/java/com/hedvig/app/service/push/managers/PaymentNotificationManager.kt

```diff
@@ -90,38 +89,38 @@ object PaymentNotificationManager {
             .notify(PAYMENT_FAILED_NOTIFICATION_ID, notification)
     }
 
-    fun sendClaimPaidNotification(context: Context, remoteMessage: RemoteMessage) {
-        // TODO: Extract parameters from remoteMessage to display more specifically how much money was paid etc
-        val pendingIntent = TaskStackBuilder
-            .create(context)
-            .run {
-                addNextIntentWithParentStack(
-                    Intent(
-                        context,
-                        LoggedInActivity::class.java
-                    )
-                )
-                getPendingIntent(0, PendingIntent.FLAG_UPDATE_CURRENT)
-            }
+    // fun sendClaimPaidNotification(context: Context, remoteMessage: RemoteMessage) {
+    //     // TODO: Extract parameters from remoteMessage to display more specifically how much money was paid etc
+    //     val pendingIntent = TaskStackBuilder
+    //         .create(context)
+    //         .run {
+    //             addNextIntentWithParentStack(
+    //                 Intent(
+    //                     context,
+    //                     LoggedInActivity::class.java
+    //                 )
+    //             )
+    //             getPendingIntent(0, PendingIntent.FLAG_UPDATE_CURRENT)
+    //         }
 
-        val notification = NotificationCompat
-            .Builder(
-                context,
-                PAYMENTS_CHANNEL_ID
-            )
-            .setSmallIcon(R.drawable.ic_hedvig_symbol_android)
-            .setContentTitle("TODO Copy")
-            .setContentText("TODO Copy")
-            .setPriority(NotificationCompat.PRIORITY_MAX)
-            .setAutoCancel(true)
-            .setChannelId(PAYMENTS_CHANNEL_ID)
-            .setContentIntent(pendingIntent)
-            .build()
+    //     val notification = NotificationCompat
+    //         .Builder(
+    //             context,
+    //             PAYMENTS_CHANNEL_ID
+    //         )
+    //         .setSmallIcon(R.drawable.ic_hedvig_symbol_android)
+    //         .setContentTitle("TODO Copy")
+    //         .setContentText("TODO Copy")
+    //         .setPriority(NotificationCompat.PRIORITY_MAX)
+    //         .setAutoCancel(true)
+    //         .setChannelId(PAYMENTS_CHANNEL_ID)
+    //         .setContentIntent(pendingIntent)
+    //         .build()
 
-        NotificationManagerCompat
-            .from(context)
-            .notify(CLAIM_PAID_NOTIFICATION_ID, notification)
-    }
+    //     NotificationManagerCompat
+    //         .from(context)
+    //         .notify(CLAIM_PAID_NOTIFICATION_ID, notification)
+    // }
```

Comment:
remove or keep?

Reply:
I wanted to keep it because I know we will implement the functionality later on.